src/components/CreateWatch.svelte

```diff
@@ -36,7 +38,9 @@ $: watchFilters = getFiltersForWatch(filter)
 
 </script>
 {#if canWatch}
-  <button class={$$props.class} data-toggle="modal" data-target="#createWatchModal">Create alert</button>
+  <button class={$$props.class} data-toggle="modal" data-target="#createWatchModal">
+      <Icon icon={faBell} />&nbsp;Create alert
```

Comment:
no need for these entities anymore FYI, just apply margin to the icon to give it space.